README.md

```diff
@@ -77,3 +79,60 @@ Spørsmål knyttet til koden eller prosjektet kan rettes mot:
 ### For NAV-ansatte
 
 Interne henvendelser kan sendes via Slack i kanalen #team-dagpenger-dev.
+
+## Vanlig oppstart av prosjektet (når alt er satt opp)
+
+1. Start opp Naisdevice (den må bli grønn)
+2. `$ git pull`
+3. `$ nvm use`
+4. `$ npm install`
+5. Bruk [trydgebrukeren](https://nav-it.slack.com/archives/C04RRM5QEAV/p1682666310232779) for å logge inn på de neste tre stegene
+6. Legg inn token fra [dp-rapportering](https://wonderwalled-azure.intern.dev.nav.no/api/obo?aud=dev-gcp.teamdagpenger.dp-rapportering) i `.env`-filen
+7. Legg inn token fra [dp-behandling](https://wonderwalled-azure.intern.dev.nav.no/api/obo?aud=dev-gcp.teamdagpenger.dp-behandling) i `.env`-filen
+8. [dp-vedtak](https://wonderwalled-azure.intern.dev.nav.no/api/obo?aud=dev-gcp.teamdagpenger.dp-vedtak)
+9. `$ npm run dev`
+10. Gå til http://localhost:3000/saksbehandling
+11. Ferdig!
+
+## Potensielle errorer
+
+### Feil node versjon
+
+```npm ERR! code EBADENGINE
+npm ERR! engine Unsupported engine
+npm ERR! engine Not compatible with your version of node/npm: execa@7.1.1
+npm ERR! notsup Not compatible with your version of node/npm: execa@7.1.1
+npm ERR! notsup Required: {"node":"^14.18.0 || ^16.14.0 || >=18.0.0"}
+npm ERR! notsup Actual: {"npm":"9.6.3","node":"v16.13.0"}
+
+npm ERR! A complete log of this run can be found in: /Use
+```
+
+Kjør `$ nvm use` og det burde løse problemet.
+
+### Manglende autentisering
```

Comment:
Litt usikker om den her er tydelig nok, manglende authentisering høres litt for generisk ut. Kanskje vi skal være mer tydelig her, eks. `Manglende autentisering ved installasjon av @navikt npm pakker` eller noe sånn?